imports/client/components/AllProfileListPage.tsx

```diff
@@ -6,13 +6,14 @@ import ProfileList from './ProfileList';
 
 const AllProfileListPage = () => {
   useBreadcrumb({ title: 'Users', path: '/users' });
-  const profilesLoading = useSubscribe('profiles');
```

Comment:
I notice that this change means that we're no longer subscribing to the roles, emails, or phoneNumber fields for a user, which means that when we pass the user list down to `ProfileList`, it will no longer be able to filter-match on the values of those fields.